Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1581,7 +1581,7 @@ PhysicalParticleContainer::AddPlasmaFlux (PlasmaInjector const& plasma_injector,
 
                 // Determine the position of the particle within the cell
                 XDim3 pos;
-                XDim3 r;
+                XDim3 r = {0._rt, 0._rt, 0._rt};
```

Comment:
Not the solution. I think we need to set r to `pos` inside `if (inject_from_eb)` or so